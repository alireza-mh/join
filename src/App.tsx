import React from 'react'

const App = () => (
  <div>
    <p>Recruiter Dashboard</p>
  </div>
)

export default App
