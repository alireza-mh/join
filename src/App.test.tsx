import React from 'react'
import { render, screen } from '@testing-library/react'

import App from './App'

test('renders', () => {
  const { container } = render(<App />)
  expect(container).toMatchSnapshot()
})
