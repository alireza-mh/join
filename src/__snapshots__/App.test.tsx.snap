// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <div>
    <p>
      Recruiter Dashboard
    </p>
  </div>
</div>
`;
